pykickstart/commands/fcoe.py

```diff
@@ -106,6 +106,9 @@ def _getArgsAsStr(self):
 class RHEL8_FcoeData(F28_FcoeData):
     pass
 
+class RHEL9_FcoeData(RHEL8_FcoeData):
```

Comment:
Shouldn't  we inherit from Fedora class (F28_FcoeData) here ? At least to be consistent with other commands created in this patch.

Reply:
Good point, fixed!